app/src/main/java/org/mifos/selfserviceapp/utils/PrefManager.java

```diff
@@ -0,0 +1,179 @@
+package org.mifos.selfserviceapp.utils;
+
+        import android.content.SharedPreferences;
+        import android.preference.PreferenceManager;
+        import android.text.TextUtils;
+
+        import com.google.gson.Gson;
+        import org.mifos.selfserviceapp.MifosSelfServiceApp;
+        import org.mifos.selfserviceapp.api.BaseURL;
+        import org.mifos.selfserviceapp.models.User;
+
+        import java.util.Set;
+
+/**
+ * @author fomenkoo for mifos android client
+ * Created by michaelsosnick on 1/22/17.
+ */
+
+public class PrefManager {
```

Comment:
@sosnickm We already have the `PrefManager`.  It's name [PreferencesHelper](https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/api/local/PreferencesHelper.java).

Remove this file and user the already present PreferencesHelper.